src/Jira/Api.php

```diff
@@ -157,14 +157,14 @@ public function getAttachment($attachmentId)
         return $result;
     }
 
-    public function getProjects()
+    public function getProjects($expand = '')
     {
-        return $this->api(self::REQUEST_GET, "/rest/api/2/project");
+        return $this->api(self::REQUEST_GET, "/rest/api/2/project",array('expand' => $expand), true);
     }
 
-    public function getProject($projectKey)
+    public function getProject($projectKey, $expand = '')
```

Comment:
I would rather make `$expand` parameter and array and then concatenate values by comma before making API call (right in this method).